client/src/components/Viewer/components/GetFiles/components/RenderFiles/RenderFile.js

```diff
@@ -274,6 +274,7 @@ function RenderFile(props) {
             info={pyramid}
             format={format}
             outer={outer}
+            email={props.email}
```

Comment:
can you add comments for this in code , as to why this new prop is added. Context needed

Reply:
added